library/src/main/java/com/airbnb/android/airmapview/MapboxWebViewMapFragment.java

```diff
@@ -22,4 +22,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
     return view;
   }
+
+  @Override public void setMapType(MapType type) {
+    // TODO: Not implemented
```

Comment:
Exactly. I couldn't find the correct method to call. Do you mind pointing me to the correct docs page, please?